src/libais/ais.h

```diff
@@ -1566,9 +1566,49 @@ class Ais8_200_10 : public Ais8 {
   Ais8_200_10(const char *nmea_payload, const size_t pad);
 };
 
-// 21 and 22 do not exist
+// vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf
+// ETA at lock/bridge/terminal
+// Todo : complete this class
+class Ais8_200_21 : public Ais8 {
+ public:
+  string country;         // UN country code                    12 bits
+  string location;        // UN location code                   18 bits
+  string section;         // Fairway section number             30 bits
+  string terminal;        // Terminal code                      30 bits
+  string hectometre;      // Fairway hectometre                 30 bits
+  int eta_month;          // ETA at lock/bridge/terminal        20 bits 
+  int eta_day;
```

Comment:
UTC or what timezone?

Reply:
According to http://www.iala-aism.org/asm/eta-at-lockbridgeterminal/, this is UTC.